src/Spryker/Glue/ProductsRestApi/Processor/AbstractProducts/AbstractProductsReader.php

```diff
@@ -141,4 +163,25 @@ protected function addConcreteProducts(RestResourceInterface $restResource, Rest
 
         return $restResource;
     }
+
+    /**
+     * @param array $abstractProductData
+     * @param \Spryker\Glue\GlueApplication\Rest\Request\Data\RestRequestInterface $restRequest
+     *
+     * @return \Spryker\Glue\GlueApplication\Rest\JsonApi\RestResourceInterface[]
+     */
+    protected function createRestResourcesFromBulkAbstractProductStorageData(array $abstractProductData, RestRequestInterface $restRequest): array
```

Comment:
I'd rather place this method before `addConcreteProducts()` method for better readability.